### PR TITLE
Update SBT to version 1.3.10 on the Debian CI.

### PR DESCRIPTION
**Pull request by phaazon**
That commit should fix the situation with failing sbt downloads.

### CODE REVIEW

**Comment by phaazon** [comment]
Looks like this doesn’t solve our problems.

**Comment by phaazon** [comment]
Looks like the `sbt` problem is now gone. However, we’re still failing with network issues with explorers… It should be fine to merge this PR though as it fixes what it intends to.

File: .circleci/setup_macos.sh [hunk 1]
```diff
@@ -22,7 +22,7 @@ if [ "$BUILD_CONFIG" == "Release" ]; then
 	brew install awscli
 	echo "========> Install sbt"
 	brew install sbt
-	sbt sbtVersion
+	#sbt sbtVersion
```
Comment (pollastri-pierre) [inline .circleci/setup_macos.sh:25]:
```suggestion
```
since we don't use it we can remove the line I think.

Comment (phaazon) [reply .circleci/setup_macos.sh:25]: Sure.

Comment (phaazon) [reply .circleci/setup_macos.sh]: Done.

**Pull request merged**